lib/internal/test_runner/test.js

```diff
@@ -560,7 +561,7 @@ class Test extends AsyncResource {
         const map = lazyFindSourceMap(this.loc.file);
         const entry = map?.findEntry(this.loc.line - 1, this.loc.column - 1);
 
-        if (entry !== undefined) {
+        if (entry !== undefined && ObjectKeys(entry).length > 0) {
```

Comment:
Handle cases when `findEntry` returns an empty object https://github.com/nodejs/node/blob/main/lib/internal/source_map/source_map.js#L213-L218

Reply:
It was crashing at https://github.com/nodejs/node/pull/55231/files#diff-42583d1c2fb89ce93a6c2922840a13f8026995468ffe8b0ac8319d1749de6acbR571 since it was setting `this.loc.file` to `undefined`.